native/acter/src/api/auth.rs

```diff
@@ -87,7 +88,12 @@ pub async fn make_client_config(
         db_passphrase,
         reset_if_existing,
     )
-    .await?;
+    .await?
+    .with_encryption_settings(EncryptionSettings {
+        auto_enable_cross_signing: true,
+        backup_download_strategy: BackupDownloadStrategy::AfterDecryptionFailure,
+        auto_enable_backups: true,
```

Comment:
This is at the heart of the fix: we needed to pass the proper configuration upon client creation or the magic of the sdk wouldn't actually sync the backup properly.